test/consts/const.go

```diff
@@ -19,4 +23,8 @@ const (
 	MetalLBConfigMapName = apply.MetalLBConfigMap
 	// DefaultOperatorNameSpace is the default operator namespace
 	DefaultOperatorNameSpace = "metallb-system"
+	// LogsExtractDuration represents how much in the past to fetch the logs from
+	LogsExtractDuration = 10 * time.Minute
+	// MetalLBTestNameSpace is the namespace used in the tests
+	MetalLBTestNameSpace = "metallb-test-namespace"
```

Comment:
Let's not leak names from the e2e test to the main package, that's probably the reason why this wasn't here in first instance.

Reply:
It's not the main package, those are the consts for metallb's tests, under the `test` dir.
For example look where we use the consts:
MetalLBOperatorDeploymentName
MetalLBOperatorDeploymentLabel
MetalLBOperatorCRDName

Reply:
Thanks for the clarification. I understand now what you mean and totally agree. will fix.